codespell_lib/data/dictionary.txt

```diff
@@ -55750,6 +55761,7 @@ thogh->though, thigh,
 thoght->thought
 thoghts->thoughts
 thonic->chthonic
+thoose->those
```

Comment:
```suggestion
thoose->those, choose,
```

Reply:
IMHO the c an t are quite far from each other and therefore thoose->choose is a bit far-fetched.

Reply:
Not on Dvorak keyboards though:

![The modern Dvorak layout (U.S. layout)](https://upload.wikimedia.org/wikipedia/commons/2/25/KB_United_States_Dvorak.svg)

Reply:
Surprising keyboard, thanks for the info.  It was added anyway.